server.js

```diff
@@ -17,8 +17,8 @@ const server = require("fastify")({
   ignoreTrailingSlash: true,
 });
 
-const storage = require("node-persist");
-const cache = require("nano-cache");
+const NanoCache = require('nano-cache');
+const cache = new NanoCache({ ttl: 30 * 60 * 1000 }); // 30 minutes in milliseconds
```

Comment:
I arbitrarily chose 30 minutes, but we can change this to whatever you like, @roosto